docs/BEMAN_STANDARD.md

```diff
@@ -422,3 +422,21 @@ copyright notice following the SPDX license identifier.
 
 **[CPP.NAMESPACE]** RECOMMENDATION: Headers in `include/beman/<short_name>/` should export
 entities in the `beman::<short_name>` namespace.
+
+## Continous Integration
+
+**[CI.PLATFORM]** REQUIREMENT: All CI should be implemented using GitHub Actions or an CI platform that appripriatly interacts with the code control system.
+
+**[CI.TEST]** REQUIREMENT: CI must execute appropriate testing of reasonable scope tagged with each commit, and must fail with test cases.
+
+**[CI.TEST.CROSS_PLATFORM]** REQUIREMENT: CI enabled tests must be executed across major platforms if appripriate.
```

Comment:
We don't have multiple dot usage in the name of the rules in this doc!

+ remove `CI.TEST.CROSS_COMPILER_FAMILIES` and `CI.TEST.CROSS_COMPILER_VERSIONS`

```suggestion
**[CI.USE_CROSS_PLATFORM_TESTS]** RECOMMANDATION: The CI tests should be executed across major platforms and compilers.
```